### PR TITLE
Reorder TOC

### PR DESCRIPTION
**Pull request by josegonzalez**
Also move Error/Exception to `General Topics`

### CODE REVIEW

**Comment by markstory** [comment]
Does this mean the CSS generated menus need to change now?

**Comment by josegonzalez** [comment]
Ah crap forgot about those. I'll go through this tonight and update everything.

**Comment by lorenzo** [comment]
bump

**Comment by cake17** [comment]
You can see the following screen shots for each language

en
es
fr
ja
pt
zh

**Comment by cake17** [comment]
Please wait a little bit, I check that Migrations title has been added in `Other` category

**Comment by cake17** [comment]
It's fine, Migrations is in the toc, see the result for en version (all good for other languages)

en

File: themes/cakephp/static/cake.css [hunk 1]
```diff
@@ -337,6 +337,18 @@ a.improve {
 #sidebar-navigation > ul > li:lang(fr):nth-of-type(1):before {
     content: "Préface";
 }
+#sidebar-navigation > ul > li:lang(es):nth-of-type(1):before {
```
Comment (antograssiot) [inline themes/cakephp/static/cake.css]: I don't see much values in including these if they are not translated as the english verison will be used anyway, and it add extra maintenance...

**Comment by antograssiot** [comment]
looks good even if the alphabetization make few sense for translations.
nice work

File: themes/cakephp/static/cake.css (hunk 1 above)
Comment (cake17) [inline themes/cakephp/static/cake.css]: Just to encourage people to translate these titles, but I can remove these if necessary

**Comment by lorenzo** [comment]
Thanks for the changes!

**Pull request merged**